PokeOneWeb/Frontend/pokeoneweb/src/app/core/components/app-version/app-version.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, OnInit, Output } from '@angular/core';
+import { environment } from 'src/environments/environment';
+import packageJson from '../../../../../package.json';
+
+@Component({
+  selector: 'pokeone-app-version',
+  templateUrl: './app-version.component.html',
+  styleUrls: ['./app-version.component.scss']
+})
+export class AppVersionComponent implements OnInit {
+  @Output() appVersionText = "";
+  containerClass = "app-version-container";
+
+  constructor() { 
+    this.appVersionText = environment.stage + ' ' + packageJson.version;
```

Comment:
If you read the version from the package.json file, the "appVersion" field is not required in the environment.ts files, right?

Reply:
Removed unnecessary "appVersion" fields. Due to the change of an other approach I forgot to delete them from all files.